src/NzbDrone.Core.Test/OrganizerTests/FileNameBuilderTests/FileNameBuilderFixture.cs

```diff
@@ -564,6 +566,27 @@ public void should_be_able_to_use_original_filename()
                    .Should().Be("30 Rock - 30 Rock - S01E01 - Test");
         }
 
+        [Test]
+        public void should_be_same_different_cultures()
+        {
+            CultureInfo[] cultures = CultureInfo.GetCultures(CultureTypes.AllCultures);
+            _movie.TmdbId = 124578;
+            _movie.Year = 2020;
+            GivenMediaInfoModel();
+
+            _namingConfig.StandardMovieFormat = "{Movie CleanTitle} ({Release Year}) [{Quality Title}] [tmdb-{TmdbId}] [{MediaInfo AudioCodec}]";
+
+            foreach (var culture in cultures)
```

Comment:
This takes 8 seconds to run, so we should narrow to just a few common and problematic cultures